### ISSUE TITLE
AzureManagedCluster E2E Tests

### ISSUE THREAD

**Issue by dtzar** [kubernetes-sigs/cluster-api-provider-azure#2873]
/kind feature
/area managedclusters

**Describe the solution you'd like**
This is a larger "Epic" feature issue to centrally track progress of all of the E2E tests missing/desired in order to help CAPZ ManagedCluster graduate out of experimental status.  

Entire backlog to graduate out of experimental status is on [this project board](https://github.com/orgs/kubernetes-sigs/projects/26/views/1).

- [x] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2826
- [x] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2854
- [ ] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2855
- [x] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2866
- [x] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2867
- [ ] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2868
- [ ] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2869
- [x] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2870
- [ ] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2871
- [ ] https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2872

**Comment by jackfrancis** [comment]
One of the key metrics for this pre-capz-1.8 release is using tests to gauge stability prior to release. That is looking good!:

- https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure#capz-periodic-e2e-aks-main

**Comment by dtzar** [comment]
We discussed the idea in community call of closing this issue out.  Since we've deprioritized the remaining E2E tests, I think it's OK to close this issue out and it can still be a reference in closed state.  

I do think it would be good @jackfrancis to look at the remaining E2E tests and consider adding good-first-issue labels if you think appropriate or closing out the test issues if you don't think they warrant doing at all anymore.
